### ISSUE TITLE
cargo trike stickers 

### ISSUE THREAD

**Issue by carolinestj** [CityOfBoston/design-team-workflow#1569]
Contact: Jaclyn Youngblood 

The frame of the bike is bright red, glossy. (Probably not exactly Freedom Trail Red, but it should be pretty noticeable. :D) (see attached, also)

I'm thinking we want four stickers:
- Front (furthest away from the bicycle frame, 26.4" x 17.3", though I defer to you if we should make them a tiny bit smaller than the size of box, so there's some wood underneath showing as trim)
- Back (closest to bicycle frame, taller height dimension, 26.4" (base), 21.3" (height); 27.6" (top line))
- Right Side (17.3" (front side edge) x 35.4" (bottom) x  26.4" (back side edge) x 39.4" (top))
- Left Side (same dimensions as right side, but flipped)

What do you think about the following?
- Front: City seal, Mayor's Name
- Back: Some sort of city image (kind of like what you did for International Women's Day a few years ago with all the different womxn iconography // I think I saw you did something for BTD, perhaps, that was kind of city-scape-y? Maybe some active transportation mode elements included?
- Right side: Some text about Climate Action Plan Update 2019, Strategy 12: Reducing municipal fleet emissions (maybe a website linking to the environment dept?)
- Left side: We're trying to come up with a name beyond "CAPtrike" for this, and thinking about maybe tying in a famous Boston woman (cyclist)'s name. but TBD. Open to suggestions for this side.

Basically, our goals with these stickers are:
1) This is a COB official vehicle
2) Teaching people about the Climate Action Plan update / carbon neutrality
3) Generating excitement and photo op at Mayor's neighborhood coffee hours, etc., things where advance shows up with equipment and could maybe cart the equipment over in this trike

the trike won't even arrive until late April or early May at the earliest...and we probably just need a week turnaround to actually print the stickers from the vendor.

![Screen Shot 2020-03-30 at 2.24.25 PM.png](https://images.zenhubusercontent.com/5ca279ab45147541ad68a704/c9665656-801c-490a-aab1-a7fff0cd0e00)![image(4).png](https://images.zenhubusercontent.com/5ca279ab45147541ad68a704/bf4610b2-3390-4791-b9d5-db42ee1b06bc)

**Comment by carolinestj** [comment]
So maybe some of these figures can be on their little gray brick sidewalk with the storefronts up to the awning (keeping everything between women's day text and the freedom trail), and keep the cyclist, the skateboarder, and the person to the right of the cyclist with a short dress and sneakers on. And maybe they on on the sides (even though they'll be partially covered by the wheels) and the logo/seal + Mayor's name go in the blank blue that is left above where the awning-text is.
And then on the front we can have the city scape from your previous email, and let's just do a placeholder for the name of the trike in big Montserrat and below it in Lora, "The City of Boston's first cargo e-tricycle".
And we can figure out where to put the one sentence fact after we have those three sides mocked up.
It may not work, but just trying to get a sense for look + feel.

**Comment by carolinestj** [comment]
1) Wonder if there's a way to make that side-graphic text pop a bit more on the blue. Does Lora bold look too heavy? Wonder if there's a way to make the contrast a bit more obvious between the text and the "sky".
2) Can we add a kid-age-looking person and maybe replace one of these lady characters with a more male-presenting character? Just thinking that although the trike is named after a woman, we can have one male-presenting person on the sticker. :)
3) I know it's going to get cluttery, but for the sake of education, can we try putting in this sentence under the logos on the nameplate? "I'm named after "Kittie" Knox, a cycling pioneer who bravely confronted gender and racial barriers in 19th-century Boston." And maybe try those logos in white? Idk. :)

Also (and maybe you already have this), can you make a duplicate of the side graphic and then just change the top angle to be "going the other way" if that makes sense? Like, these dimensions work on the first side, but then for the "second side" it needs to have the angle part going the other way (otherwise the sticker will have to go on the inside of the second side, which doesn't work)
